samples/GuestConfiguration/installed-application-linux/README.md

```diff
@@ -9,7 +9,7 @@ Included policy definitions:
 
 - audit:
   - ID: `/providers/Microsoft.Authorization/policyDefinitions/fee5cb2b-9d9b-410e-afe3-2902d90d0004`
-  - Source: [policy definition](./audit/)
```

Comment:
@DCtheGeek how do you want to handle this change since there is a sample on the docs page?  Right now the old "audit" content isn't being deleted yet but the rest of the content is changing to not reference it.

Reply:
As long as the file isn't being removed, we are fine for now.  Per the other comment, we'll be removing the samples "per RP" from docs eventually and referencing them here instead.